docs/source/start.ipynb

```diff
@@ -37,7 +38,36 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "X, y = evalml.demos.load_breast_cancer()"
+    "X, y = evalml.demos.load_fraud(n_rows=1000, return_pandas=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, let's clean the data. Since EvalML accepts a pandas input, it can run type inference on this data directly. If you\u2019d like to change the types inferred by EvalML, you can use the `infer_feature_types` utility method. Here's what we're going to do with the following dataset:\n",
```

Comment:
Really nit-picky / we don't do a great job at this but for readability, can we stick to a more consistent POV 😝  Either second or third is fine but I think it reads easier than jumping back and forth